packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -148,7 +148,12 @@ export const Workbench: React.FunctionComponent<{
 
   const revealAttachment = React.useCallback((attachment: AfterActionTraceEventAttachment) => {
     selectPropertiesTab('attachments');
-    setRevealedAttachment(attachment);
+    setRevealedAttachment(currentValue => {
```

Comment:
It looks to me this can be reverted.

Reply:
Nope, this is required that to re-trigger the animation. We're discussing the specific form in https://github.com/microsoft/playwright/pull/34353#discussion_r1918994918. We don't need it in the HTML reporter because we have the `URLSearchParams` context, but that's not a thing in the trace viewer.